README.md

```diff
@@ -1,330 +1,107 @@
-  },
-  {
-    "flower_id": "flower_02",
-    "width": 7,
-    "length": 2,
-    "color": "white",
-    "species": "lily"
-  }
-]
-```
-<br />
-
-#### Text dataset format
-
-- dataset must have an **ID column** (`review_id` in the example below) - a column containing identifiers that uniquely identify each row.
-- dataset must have a **text column** (`review` in the example below) that serves as the sole predictive feature for modeling the label and identifying label issues.
-- dataset must have a **label column** (`sentiment` in the example below) which you either want to train models to predict or simply find erroneous values in.
-- Apart from the reserved column name: `clean_label`, you are free to name the columns in your dataset in any way you want.
-
-###### .csv, .xls/.xlsx
-
-| review_id | review | sentiment |
-|:----------|:-------|-----------|
-| review_1  | The sales rep was fantastic!     | positive  |
-| review_2  | He was a bit wishy-washy.     | negative  |
-
-###### .json
-
-```json
-[
-  {
-    "review_id": "review_1",
-    "review": "The sales rep was fantastic!",
-    "label": "positive"
-  },
-  {
-    "review_id": "review_2",
-    "review": "He was a bit wishy-washy.",
-    "label": "negative"
-  }
-]
-```
-<br />
-
-#### Image dataset format
-- Image datasets have two components:
-  - **Collection of image files.**
-  - **Labels file** - A mapping from each image filepath to a class label. This mapping can be supplied either in a .csv, .xls/.xlsx, or .json format.
-
-###### Labels file format
-- must have an **ID column** (`vizzy_id` in the example below) - a column containing identifiers that uniquely identify each row.
-- must have a **filepath column** (`vizzy_path` in the example below) that contains relative path to the image file.
-- must have a **label column** (`label` in the example below) that contains the label for the corresponding image file.
-- may have any number of extra metadata columns that will not be used to model labels and identify label issues. Apart from the reserved column name: `clean_label`, you are free to name these columns any way you want.
-
-###### Dataset format
-
-###### .csv, .xls/.xlsx
-
-| vizzy_id | vizzy_path | label |
-|:----------|:-------|-----------|
-| 1  | Dataset/scruppy.jpeg    | cat  |
-| 2  | Dataset/tuffy/fluffy.png    | cat  |
-| 3  | oreo.jpeg    | dog  |
-| 4  | Dataset/mocha/mocha.jpeg    | dog  |
-
-###### .json
-
-```json
-[
-  {
-    "vizzy_id": "1",
-    "vizzy_path": "Dataset/scruppy.jpeg",
-    "label": "cat"
-  },
-  {
-    "vizzy_id": "2",
-    "vizzy_path": "Dataset/tuffy/fluffy.png",
-    "label": "cat"
-  },
-  {
-    "vizzy_id": "3",
-    "vizzy_path": "oreo.jpeg",
-    "label": "dog"
-  },
-  {
-    "vizzy_id": "4",
-    "vizzy_path": "Dataset/mocha/mocha.jpeg",
-    "label": "dog"
-  }
-]
+# download your cleanset or apply corrections to your local Pandas or PySpark dataset!
+# @ANISH !!! how do we want to tell users to find their cleanset ID?
```

Comment:
Why not point people to the export cleanset button? (And in particular, the export via CLI tab?)

If that tab needs to be improved, we can improve the UI/text there.